### ISSUE TITLE
Not enough label packed in img_list or rec file.

### ISSUE THREAD

**Issue by dingqunfei** [zhreshold/mxnet-ssd#195]
i use ssd on my dataset with 4 classes.
Fisrt,getting file.rec:
python im2rec.py --list  --train-ratio 0.8 --recursive  wm323  /root/userfolder/data323
python im2rec.py --resize 300 --quality 100 --num-thread 8 --pass-through wm323 /root/userfolder/data323

the file.lst shows :
2849	1.000000	elong/280002039.jpg
2804	1.000000	elong/280001954.jpg
3525	2.000000	qunar/30103.jpg
3956	3.000000	qyer/401015.jpg
482	0.000000	ctrip/102034.jpg
3982	3.000000	qyer/401041.jpg
571	0.000000	ctrip/102144.jpg
3390	1.000000	elong/280004231.jpg
1389	0.000000	ctrip/18000700.jpg
362	0.000000	ctrip/101368.jpg

then, training with cmd:
python train.py --train-path wm323_train.rec --val-path wm323_val.rec --network vgg16_ssd_300 --epoch 1 --gpus 2,3 --num-class 4 --batch-size 128 --lr 0.001 --class-name ctrip,elong,qunar,qyer

however, in the beginning,there is a error:
[10:39:48] src/io/iter_image_det_recordio.cc:281: ImageDetRecordIOParser: wm323_train.rec, use 48 threads for decoding..
terminate called after throwing an instance of 'dmlc::Error'
  what():  [10:39:48] src/io/iter_image_det_recordio.cc:315: Not enough label packed in img_list or rec file.
......
Aborted (core dumped)

the pretrained model is vgg16_reduced_300*300 in this repo.
what should i do?
Thanks!

**Comment by jiangziya** [comment]
@dingqunfei did you solve the problem，I have the same problem.
